### PR TITLE
Form label alignment

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

File: pkg/shell/cockpit-networking.js [hunk 1]
```diff
@@ -2700,10 +2700,11 @@ PageNetworkBondSettings.prototype = {
             }
         }
 
+        /* TODO: Migrate away from jQuery spaggheti */
```
Comment (dperpeet) [inline pkg/shell/cockpit-networking.js]: -> spaghetti

File: pkg/shell/shell.html [hunk 2]
```diff
@@ -2104,7 +2148,7 @@ <h4 class="modal-title">Run Image</h4>
                 </td>
               </tr>
               <tr id="containers-run-image-memory">
-                <td translatable="yes">Memory limit</td>
+                <td><label class="control-label" translatable="yes">Memory limit</label></td>
```
Comment (dperpeet) [inline pkg/shell/shell.html]: Most other labels are now linked to checkboxes - for consistency, so should this one.

File: pkg/shell/shell.html [hunk 3]
```diff
@@ -2120,7 +2164,7 @@ <h4 class="modal-title">Run Image</h4>
                 </td>
               </tr>
               <tr id="containers-run-image-cpu">
-                <td translatable="yes">CPU priority</td>
+                <td><label class="control-label" translatable="yes">CPU priority</label></td>
```
Comment (dperpeet) [inline pkg/shell/shell.html]: Most other labels (in run image dialog) are now linked to checkboxes - for consistency, so should this one.

File: pkg/shell/cockpit-networking.js (hunk 1 above)
Comment (dperpeet) [inline pkg/shell/cockpit-networking.js]: possibly add comment about adding linking code between labels and input controls when this is cleaned up

File: pkg/shell/cockpit-networking.js [hunk 4]
```diff
@@ -2848,10 +2849,11 @@ PageNetworkBridgeSettings.prototype = {
             max_age_input.parents("tr").toggle(options.stp);
         }
 
+        /* TODO: Migrate away from jQuery spaghetti */
```
Comment (dperpeet) [inline pkg/shell/cockpit-networking.js]: possibly add comment about adding linking code between labels and input controls when this is cleaned up

File: pkg/shell/cockpit-networking.js [hunk 5]
```diff
@@ -2968,22 +2970,23 @@ PageNetworkBridgePortSettings.prototype = {
             options.hairpin_mode = hairpin_mode_input.prop('checked');
         }
 
+        /* TODO: Migrate away from jQuery spaghetti code */
```
Comment (dperpeet) [inline pkg/shell/cockpit-networking.js]: possibly add comment about adding linking code between labels and input controls when this is cleaned up

File: pkg/shell/cockpit-networking.js [hunk 6]
```diff
@@ -3091,21 +3094,22 @@ PageNetworkVlanSettings.prototype = {
                 parent_choices.push({ title: i.Name, choice: i.Name });
         });
 
+        /* TODO: Migrate away from jQuery spaghetti */
```
Comment (dperpeet) [inline pkg/shell/cockpit-networking.js]: possibly add comment about adding linking code between labels and input controls when this is cleaned up

**Comment by dperpeet** [comment]
The dialog to create a new account doesn't look consistent, but I guess we can ignore that since we'll merge the new version soon.

File: pkg/shell/cockpit-networking.js (hunk 1 above)
Comment (stefwalter) [inline pkg/shell/cockpit-networking.js]: Fixes spelling. Needs to be completelty rewritten to use templates.

File: pkg/shell/shell.html (hunk 2 above)
Comment (stefwalter) [inline pkg/shell/shell.html]: Out of scope of the issue being fixed here. Feel free to do this as a follow up.

File: pkg/shell/shell.html (hunk 3 above)
Comment (stefwalter) [inline pkg/shell/shell.html]: Out of scope of the issue being fixed here. Feel free to do this as a follow up.

File: pkg/shell/cockpit-networking.js (hunk 4 above)
Comment (stefwalter) [inline pkg/shell/cockpit-networking.js]: Needs to be completely rewritten using templates, moved out the shell, etc.

File: pkg/shell/cockpit-networking.js (hunk 5 above)
Comment (stefwalter) [inline pkg/shell/cockpit-networking.js]: Needs to be completely rewritten using templates, moved out the shell, etc.

File: pkg/shell/cockpit-networking.js (hunk 6 above)
Comment (stefwalter) [inline pkg/shell/cockpit-networking.js]: Needs to be completely rewritten using templates, moved out the shell, etc.

**Comment by stefwalter** [comment]
Fixed issues and rebased. Most of the items are out of scope for this pull request.

**Comment by stefwalter** [comment]
I don't agree with blocking this on everything else. There's key work that this is holding back.

**Comment by dperpeet** [comment]
@stefwalter can you rebase this? conflict with current master

**Comment by stefwalter** [comment]
Rebased.

**Pull request closed without merging**